kolibri/plugins/utils/__init__.py

```diff
@@ -481,7 +481,12 @@ def check_plugin_config_file_location(version):
 def iterate_plugins():
     # Use to dedupe plugins
     plugin_ids = set()
-    for entry_point in entry_points().get("kolibri.plugins", []):
+    eps = (
+        entry_points().get("kolibri.plugins", [])
+        if type(entry_points()) == dict
+        else entry_points().select(group="kolibri.plugins")
+    )
+    for entry_point in eps:
```

Comment:
I don't know if this is a real concern, but might be nicer not to invoke `entry_points()` twice here.

Can do:
```suggestion
    all_entry_points = entry_points()
    eps = (
        all_entry_points.get("kolibri.plugins", [])
        if type(all_entry_points) == dict
        else all_entry_points.select(group="kolibri.plugins")
    )
    for entry_point in eps:
```

I don't know, but I suspect that will save a small amount of time :)